typings/config.d.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import * as Gatherer from '../lighthouse-core/gather/gatherers/gatherer.js';
+
+declare global {
+  module LH {
+    /**
+     * The full, normalized Lighthouse Config.
+     */
+    export interface Config {
+      settings: Config.Settings;
+      passes: Config.Pass[];
+    }
+
+    module Config {
+      /**
+       * The pre-normalization Lighthouse Config format.
+       */
+      export interface Json {
+        settings?: SettingsJson;
```

Comment:
shall we flesh this out a bit more?

```ts
settings?; SettingsJson;
passes?: PassJson[];
extends?: string;
audits?: string[];
categories?: Object;
groups?: Object;
```

Reply:
yeah, I think we're going to have to face this when we add types to config.js and do it right